blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnable.java

```diff
@@ -48,6 +49,7 @@
     private ScheduleContext scheduleCtx;
     private long serverTime;
     private static final Timer rollupLocatorExecuteTimer = Metrics.timer(RollupService.class, "Locate and Schedule Rollups for Slot");
+    private static final Histogram locatorsPerShard = Metrics.histogram(RollupService.class, "Locators Per Shard");
```

Comment:
This class seems to be used by all rollup related metrics. So I was going to keep it that.